tests/compile/fused_no_numpy.pyx

```diff
@@ -0,0 +1,10 @@
+# cython: test_fail_if_c_code_has = __Pyx_ImportNumPyArrayTypeIfAvailable
```

Comment:
Should we have a test for the opposite case? I don't think it's particularly useful for any except that if the fused dispatch code gets refactored then this test becomes irrelevant and we'd quite like to know that